lib/server/chat-llm.service.ts

```diff
@@ -61,14 +61,18 @@ class ChatLLMService {
     // await this.assertEnoughCredits(accountId);
 
     // retrieve the chat settings
-    const settings = await chatMessagesService.getChatSettings(referenceId, address as Address);
+    const settings = await chatMessagesService.getChatSettings(
+      referenceId,
+      address as Address,
+    );
     const systemMessage = settings.systemMessage;
     const maxTokens = settings.maxTokens;
+    const tools = settings.tools;
 
     // we need to limit the history length so not to exceed the max tokens of the model
     // let's assume for simplicity that all models have a max tokens of 4096
     // so we need to make sure that the history doesn't exceed output length + system message length
-    const maxModelTokens = 128000;
+    const maxModelTokens = 228000;
```

Comment:
I believe the max tokens in gpt-4o is `128000`
- Where are you seeing docs showing this value is valid for `gpt-4o`?

Reply:
I've just saw in the code that ziad recommended me.